DemoApp/DemoApp/TMButtonPage.xaml

```diff
@@ -4,189 +4,129 @@
+                </StackLayout>
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto"/>
+                    </Grid.RowDefinitions>
+                    <Label Grid.Column="0" Grid.Row="0" Text="IsDisabled"  HorizontalOptions="Center" VerticalOptions="Center"/>
+                    <Switch Grid.Column="1" Grid.Row="0" x:Name="isDisabled" IsToggled="{Binding IsDisabled}" Toggled="isDisabled_Toggled" />
+                </Grid>
+            <Label Text="Floating Buttons"></Label>
+            <StackLayout Orientation="Horizontal" Spacing="10">
                 <Modus:TMButton 
-                    x:Name="ibutton12"
-                    Title = "Large"
-                    IsDisabled="True"
-                    Command="{Binding MyCommand}"
+                    x:Name="floating"
+                    Title = "Floating Button"
+                     IsFloatingButton="True"
+                    ButtonColor="Primary"
                     IconSource="icon.png"
-                    CommandParameter="Button5"
-                    Size="Large"
-                    />
-                
-                <Modus:TMButton 
-                    x:Name="disbutton"
-                    Padding="10"
                     Command="{Binding MyCommand}"
-                    IconSource="icon.png"
-                    IsFloatingButton="True"
                     CommandParameter="Button5"
-                    IsDisabled="True"
-                    Size="XSmall"
                   />
-                </FlexLayout>
-                <Label Text="Customised Buttons"/>
-                <FlexLayout Margin="0,0,0,10">
-                    <Modus:TMButton 
-                    x:Name="disbutton2"
-                    Title = "Custom"
-                    Radius="50"
-                    Command="{Binding MyCommand}"
-                    IconSource="icon.png"
+                <Modus:TMButton 
+                    x:Name="floatingi"
                     IsFloatingButton="True"
-                    Padding="10"
-                    CommandParameter="Button5"
-                    IsDisabled="False"
-                    Size="XSmall"
-                  />
-
-
-                <Modus:TMButton  
-                    Style="{StaticResource Gold}"  
-                    Title="Hello" 
-                    Padding="10"
-                    IconSource="greenicon.png" 
-                   />
-
-                <Modus:TMButton  
-                    Style="{StaticResource Red}"  
-                    Title="Hello" 
-                    Padding="10"
+                    ButtonColor="Primary"
                     IconSource="icon.png"
```

Comment:
Icon names should be proper naming conversion. Please rename the icon.png with valid name

Reply:
Sure